src/dials/util/multi_dataset_handling.py

```diff
@@ -76,7 +76,8 @@ def sort_tables_to_experiments_order(reflection_tables, experiments):
     exp_id_to_table_idx = {}
     for i, table in enumerate(reflection_tables):
         id_values = table.experiment_identifiers().values()
-        if id_values:
+        any_blanks = any([len(id_value) == 0 for id_value in id_values])
```

Comment:
Suggest `all()` not `any()` because if you have _some_ id's this may not behave as you expect?

Reply:
Thanks!